osu.Game/Overlays/Profile/Header/PreviousUsernamesContainer.cs

```diff
@@ -0,0 +1,175 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Input.Events;
+using osu.Game.Graphics;
+using osu.Game.Users;
+using osuTK;
+using System;
+using System.Linq;
+
+namespace osu.Game.Overlays.Profile.Header
+{
+    public class PreviousUsernamesContainer : CompositeDrawable
```

Comment:
You call this class a container, but it does not derive from `Container` class. This is misleading.

Reply:
Although it's kind-of a container, un-modifiable container?

Reply:
I'm open for suggestions